templates/mojave-xcode.json

```diff
@@ -56,9 +56,11 @@
     },
     {
       "inline": [
-        "brew install xctool"
+        "brew install xctool",
+        "brew install gcc",
```

Comment:
Oh. Actually `mojave-xcode` is based of `mojave-base` so you need to install them only once in `mojave-base`. You can revert changes to `mojave-xcode.json`